versions.props

```diff
@@ -28,8 +28,8 @@ com.github.ben-manes.caffeine:caffeine = 3.1.1
 
 # test deps
 junit:junit = 4.13.2
-org.junit.jupiter:* = 5.8.2
-org.junit.vintage:* = 5.8.2
+org.junit.jupiter:* = 5.9.1
+org.junit.vintage:* = 5.9.1
```

Comment:
I think we're going to have to pin this until https://github.com/palantir/dialogue/pull/1815 merges